.idea/workspace.xml

```diff
@@ -2,6 +2,10 @@
 <project version="4">
   <component name="ChangeListManager">
     <list default="true" id="136454cb-c884-4d73-a64a-d5c90e8ae1e9" name="Default Changelist" comment="">
+      <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/workspace.xml" afterDir="false" />
```

Comment:
Can you please investigate what is wrong in `gitigrone` ? Please remove this files from git.